content/docs/SDKs/python/changelog.mdx

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
remove this file. we already have a python changelog which you correctly updated